src/shared/components/search-result/search-result.component.ts

```diff
@@ -320,7 +320,7 @@ export class SearchResultComponent implements OnChanges, OnInit {
     let condition = false;
     if (this.consumerSortingEnabled) {
       const isColumn = column.case_field_id === this.consumerSortParameters.column;
-      const isDescending = this.consumerSortParameters.order === SortOrder.DESCENDING;
+      const isDescending = this.consumerSortParameters.order === SortOrder.ASCENDING;
```

Comment:
This leads to very unreadable code, as 'isDescending' now is true when it's sorted ascending. Either change where you need it ascending by using !this.isDescending, or change the name of the var to isAscending.

Reply:
You're right. I've changed the name now.